CalculateCalendarLogic/CalculateCalendarLogic.swift

```diff
@@ -20,8 +20,8 @@ public enum Weekday: Int {
     case sun, mon, tue, wed, thu, fri, sat
 
     init?(year: Int, month: Int, day: Int) {
-        let cal = Calendar.current as NSCalendar
-        guard let date = cal.date(era: AD, year: year, month: month, day: day, hour: 0, minute: 0, second: 0, nanosecond: 0) else { return nil }
+        let cal = Calendar(identifier: .gregorian)
+        guard let date = cal.date(from: DateComponents(era: AD, year: year, month: month, day: day, hour: 0, minute: 0, second: 0, nanosecond: 0)) else { return nil }
```

Comment:
__[SwiftLint]__

Line should be 120 characters or less: currently 166 characters


<!-- Comment by [Sider](https://sider.review) -->